resources/js/Pages/Dashboard/Timesheet/Index.vue

```diff
@@ -197,33 +197,11 @@
               </div>
             </div>
 
-            <!-- daily total: monday -->
-            <div class="tc pv2 dtc bt bl bb bb-gray f7 gray">
-              {{ formatTime(dailyStats[0]) }}
-            </div>
-            <!-- daily total: tuesday -->
-            <div class="tc pv2 dtc bt bl bb bb-gray f7 gray">
-              {{ formatTime(dailyStats[1]) }}
-            </div>
-            <!-- daily total: wednesday -->
-            <div class="tc pv2 dtc bt bl bb bb-gray f7 gray">
-              {{ formatTime(dailyStats[2]) }}
-            </div>
-            <!-- daily total: thursday -->
-            <div class="tc pv2 dtc bt bl bb bb-gray f7 gray">
-              {{ formatTime(dailyStats[3]) }}
-            </div>
-            <!-- daily total: friday -->
-            <div class="tc pv2 dtc bt bl bb bb-gray f7 gray">
-              {{ formatTime(dailyStats[4]) }}
-            </div>
-            <!-- daily total: saturday -->
-            <div class="tc pv2 dtc bt bl bb bb-gray off-days f7 gray">
-              {{ formatTime(dailyStats[5]) }}
-            </div>
-            <!-- daily total: sunday -->
-            <div class="tc pv2 bt bl bb bb-gray off-days f7 gray">
```

Comment:
@djaiss The last cell missed the `dtc` class !

Reply:
good catch!